atomic_reactor/cli/parser.py

```diff
@@ -57,6 +57,13 @@ def parse_args(args: Optional[Sequence[str]] = None) -> dict:
     )
     source_build.set_defaults(func=task.source_build)
 
+    job = tasks.add_parser(
+        "job",
+        help="run a remote host job",
+        description="Run a remote host job.",
+    )
+    job.set_defaults(func=task.job)
```

Comment:
Based on the design, I believe `job` was intended to be a new top-level subparser, rather than a subcommand under the `task` subparser.
```shell
atomic-reactor task job
```
:x: 
```shell
atomic-reactor job remote-hosts-unlocking
```
:heavy_check_mark: 

Related to that: Jobs should not use the existing Task infrastructure